routers/web/projects/workflows.go

```diff
@@ -0,0 +1,624 @@
+					if label, ok := labelInterface.(string); ok && label != "" {
+						actions = append(actions, project_model.WorkflowAction{
+							Type:  project_model.WorkflowActionTypeAddLabels,
+							Value: label,
+						})
+					}
+				}
+			}
+		case "remove_labels":
+			// Handle both []string and []any from JSON unmarshaling
+			if labels, ok := value.([]string); ok && len(labels) > 0 {
+				for _, label := range labels {
+					if label != "" {
+						actions = append(actions, project_model.WorkflowAction{
+							Type:  project_model.WorkflowActionTypeRemoveLabels,
+							Value: label,
+						})
+					}
+				}
+			} else if labelInterfaces, ok := value.([]any); ok && len(labelInterfaces) > 0 {
+				for _, labelInterface := range labelInterfaces {
+					if label, ok := labelInterface.(string); ok && label != "" {
+						actions = append(actions, project_model.WorkflowAction{
+							Type:  project_model.WorkflowActionTypeRemoveLabels,
+							Value: label,
+						})
+					}
+				}
+			}
+		case "issue_state":
+			if strValue, ok := value.(string); ok {
+				v := strings.ToLower(strValue)
+				if v == "close" || v == "reopen" {
+					actions = append(actions, project_model.WorkflowAction{
+						Type:  project_model.WorkflowActionTypeIssueState,
+						Value: v,
+					})
+				}
+			}
+		}
+	}
+
+	return actions
+}
+
+func WorkflowsEvents(ctx *context.Context) {
+	projectID := ctx.PathParamInt64("id")
+	p, err := project_model.GetProjectByID(ctx, projectID)
+	if err != nil {
+		if project_model.IsErrProjectNotExist(err) {
+			ctx.NotFound(nil)
+		} else {
+			ctx.ServerError("GetProjectByID", err)
+		}
+		return
+	}
+	if p.Type == project_model.TypeRepository && p.RepoID != ctx.Repo.Repository.ID {
+		ctx.NotFound(nil)
+		return
+	}
+	if (p.Type == project_model.TypeOrganization || p.Type == project_model.TypeIndividual) && p.OwnerID != ctx.ContextUser.ID {
+		ctx.NotFound(nil)
+		return
+	}
+
+	workflows, err := project_model.FindWorkflowsByProjectID(ctx, projectID)
+	if err != nil {
+		ctx.ServerError("FindWorkflowsByProjectID", err)
+		return
+	}
+
+	type WorkflowConfig struct {
+		ID            int64                                   `json:"id"`
+		EventID       string                                  `json:"event_id"`
+		DisplayName   string                                  `json:"display_name"`
+		WorkflowEvent string                                  `json:"workflow_event"` // The workflow event
+		Capabilities  project_model.WorkflowEventCapabilities `json:"capabilities"`
+		Filters       []project_model.WorkflowFilter          `json:"filters"`
+		Actions       []project_model.WorkflowAction          `json:"actions"`
+		Summary       string                                  `json:"summary"` // Human readable filter description
+		Enabled       bool                                    `json:"enabled"`
+		IsConfigured  bool                                    `json:"isConfigured"` // Whether this workflow is configured/saved
```

Comment:
Why sometimes `display_name`, sometimes `isConfigured`?

Reply:
These are two columns. I don't understand what you mean.